tests/unit/api/objectReplicationMD.js

```diff
@@ -114,6 +122,92 @@ function createBucketWithReplication(hasStorageClass) {
     });
 }
 
+// Initiate an MPU, put a part with the given body, and complete the MPU.
+function putMPU(key, body, cb) {
+    return async.waterfall([
```

Comment:
Unit tests have to use dummy data, otherwise these might as well be a functional tests. You can setup the mpu bucket and part data in `metadata.keyMaps` and when you do completeMultipartUpload it would "just work".